content/applications/productivity/discuss/calls/sfu_server.rst

```diff
@@ -0,0 +1,61 @@
+==========
+SFU server
+==========
+
+In Odoo Discuss, the Selective Forwarding Unit (SFU) server enables efficient handling of calls with
+many participants. Unlike peer-to-peer connections which can become resource-intensive with multiple
+users, the SFU server receives media streams from each participant and selectively forwards them to
+others, significantly improving call quality and scalability. Its sources can be found in the
+`odoo/sfu repository <https://github.com/odoo/sfu>`_.
+
+.. note::
+   In Odoo.sh and Odoo SaaS, the SFU servers are already provided. Adding your own in the
+   settings will override the default one.
+
+What is a Selective Forwarding Unit (SFU)?
+------------------------------------------
+
+An SFU server is a type of media server that receives audio and video streams from each participant
+in a call and selectively forwards them to other participants. This approach is more efficient than
+peer-to-peer connections, especially in calls with many participants.
+
+.. image:: sfu_server/sfu-diagram.png
+   :align: center
+   :alt: Diagram of p2p vs sfu connection layout
+
+.. image:: sfu_server/sfu-graph-scaling.png
+   :align: center
+   :alt: Graph showing the bandwidth scaling of p2p vs sfu
```

Comment:
Do we have the rights to use these pictures?

Reply:
Yes because I made them